content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -324,8 +324,8 @@ Please note that:
 
 1. To make dynamic kubelet configuration work, flag `--dynamic-config-dir=/var/lib/kubelet/config/dynamic` should be specified
    in `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`
-1. Kubelet init configuration can be changed by using kubeadm MasterConfiguration file by setting `.kubeletConfiguration.baseConfig`.
-   See [using kubeadm init with a configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) for more detail
+1. The kubelet configuration can be changed by passing a `KubeletConfiguration` object to `kubeadm init` or `kubeadm join` by using
```

Comment:
s/KubeletConfiguration/NodeRegistrationOptions ?

Reply:
i can add that.
i wish we had a doc that talks about config only and covers facts such as the multiple configs separated with `---` . :\

Reply:
@fabriziopandini updated.